.circleci/config.yml

```diff
@@ -0,0 +1,91 @@
+version: 2.1
+
+commands:
+  docker-pull:
+    parameters:
+      image:
+        type: string
+    steps:
+      - run: docker pull <<parameters.image>>
+  docker-run:
+    parameters:
+      image:
+        type: string
+      command:
+        type: string
+    steps:
+      - run: docker run -e PLATFORM -v $(pwd):/test <<parameters.image>>  /bin/bash -c 'cd /test;<<parameters.command>>'
+
+jobs:
+  linux-arm64:
+    machine:
+      image: ubuntu-2004:202111-01
```

Comment:
What is this image?

Reply:
circleci releases images quarterly. This is their equivalent of ubuntu-latest, but they don't have an alias that we can use to always stay on latest.

Reply:
Ok. Follow up task to file an issue to write a GHA job to check for new images? the way we do for boringssl

Reply:
Since they release these quarterly the task maybe is a GHA job to file an issue to remind us every 3 months? 😂 I don't think they have any API to fetch these, but whoever tackles this should determine that first

Reply:
Sure, that's fine :-) As a starting point, let's just file an issue to figure out how.